modules/farm_fd2_school/src/entrypoints/Vue4/App.vue

```diff
@@ -131,14 +138,21 @@ export default {
       pickedPlant: -1,
       cropList: ['ARUGULA', 'ASPARAGUS', 'BEAN', 'RADISH'],
       plantList: [
-        { id: 1, date: '04/02/2019', location: 'D', bed: '' },
-        { id: 2, date: '04/02/2019', location: 'GHANA', bed: 'GHANA-2' },
-        { id: 3, date: '04/02/2019', location: 'GHANA', bed: 'GHANA-4' },
+        { id: 1, date: '06/02/2019', location: 'D', bed: '' },
+        { id: 2, date: '04/05/2019', location: 'GHANA', bed: 'GHANA-2' },
+        { id: 3, date: '07/14/2019', location: 'GHANA', bed: 'GHANA-4' },
         { id: 4, date: '06/05/2019', location: 'GHANA', bed: 'GHANA-4' },
       ],
       unitList: ['BUNCH', 'EACH', 'POUND'],
     };
   },
+  computed: {
+    sortedPlantList() {
+      return [...this.plantList].sort((a, b) => {
+        return this.parseDate(b.date) - this.parseDate(a.date);
```

Comment:
This looks like it works, but there are some simpler ways of accomplishing the same thing. For example:
```
    sortedPlantList() {
      return [...this.plantList].sort(
        (a, b) => new Date(a.timestamp) - new Date(b.timestamp)
      );
    },
```